backend/fms_core/templates.py

```diff
@@ -515,7 +533,23 @@
       },
   ],
   "user prefill info": {
+      # borrowed from transfer template
+      "Destination Container Kind": [
+        "Tube",
```

Comment:
you should cast SAMPLE_CONTAINER_KINDS from containers.py into a list instead. The less places we have to update the same information the best.